### ISSUE TITLE
Outside label on PieSeries does not allow the label to be split over 2 lines

### ISSUE THREAD

**Issue by ArvindSJoshi** [oxyplot/oxyplot#1181]
I tried adding "\n" in the OutsideLabelFormat for a PieSeries. However, the \n is ignored. I need to be able to wrap the label because the label I have is too long and it forces me to make the Pie much smaller than I want to.

**Comment by dodikk** [comment]
@ArvindSJoshi , what is your target platform? In case it is "Microsoft Windows", have you tried `\r\n` (`CR+LF`) which is a default for the given platform?

https://blog.codinghorror.com/the-great-newline-schism/

P.S. Just a guess.

**Comment by ArvindSJoshi** [comment]
I am using OxyPlot on Xamarin Forms app I have tried all the usual thing like \n\r, \n\n, Environment.newline etc. Nothing is working. As per the documentation label formats for many other charts mention \n. However, for PieSeries it is not working.

**Comment by dodikk** [comment]
> I am using OxyPlot on Xamarin Forms app 

I was asking if the issue reproduced on all platforms. (an alternative is "iOS only"/"Android only"/"windows only", etc.). Meaning...

```
Which platform or platforms are you testing on?
```

> I have tried all the usual thing like \n\r, \n\n, Environment.newline etc. Nothing is working.

This leads to the conclusion that the issue reproduces on all platforms (just my assumption). Please correct if I am wrong.

**Comment by objorke** [comment]
I think it should be possible to use the `DrawMultilineText` method in the `PieSeries`

- https://github.com/oxyplot/oxyplot/blob/develop/Source/OxyPlot/Rendering/RenderContext/RenderingExtensions.cs#L569
- https://github.com/oxyplot/oxyplot/blob/develop/Source/OxyPlot/Series/PieSeries.cs#L356

**Comment by ArvindSJoshi** [comment]
So I should create my own version of the code and try invoking DrawMultiLineText inside PieSeries?

**Comment by objorke** [comment]
yes, you can derive a new class from the `PieSeries` and override the whole ´Render´ method

**Comment by ArvindSJoshi** [comment]
I could get it to work with DrawMultilineText. Thanks objorke. I replaced the Oxyplot.dll in the Oxyplot portable-net45 folder in my Packages folder. Is that the right way or is there some way I can create my own package and install it?

**Comment by ArvindSJoshi** [comment]
This is not working if I derive a CustomPieSeries class from PieSeries, use DrawMultiLineText method in Render and include the CustomPieSeries class into my PCL. I don't get an error but nothing renders. Can someone please help?

**Comment by alexhopeoconnor** [comment]
Why when I override PieSeries and try to add it to a PlotModel will it render nothing? Looking through the source code I can't see any special references to PieSeries in any of the render methods for WPF etc... so why does a custom series not get rendered?

**Comment by alexhopeoconnor** [comment]
> Why when I override PieSeries and try to add it to a PlotModel will it render nothing? Looking through the source code I can't see any special references to PieSeries in any of the render methods for WPF etc... so why does a custom series not get rendered?

Figured out you need to copy the whole PieSeries class source code as it won't render unless it inherits directly from ItemsSeries for whatever reason... Tried to find where in the code they are checking the inheritance but could not easily spot it.
